### PR TITLE
implement charts for SNX Supply and TotalIssuedSynths

### PR DESCRIPTION
**Pull request by KillerByte**
these have been added at the request of the spartan council, specifically for monitoring L2 status

It includes 2 new area charts in the `Synths` section: `SNX Supply` and `Total Issued Synths`. Both of which are able to show Year, Month, and Week.

Also the `Synths` section is enabled for L2, which displays other useful statistics which are mostly ready for display.

Note that the error seen on page load will go away once the global-debt
subgraph for mainnet finishes re-syncing (as there was a change for globl debt recently).

### CODE REVIEW

**Comment by vbstreetz** [comment]
- [ ] Missing i18n entry
Screenshot 2021-09-04 at 11 30 34

File: translations/en.json [hunk 1]
```diff
@@ -155,9 +155,17 @@
 		"title": "TOTAL ACTIVE STAKERS",
 		"infoData": "The number of total active stakers is obtained from the \"TotalActiveStaker\" entity from the <linkText>Synthetix subgraph.</linkText> The chart data shows the \"TotalDailyActiveStaker\" entity over time."
 	},
-	"total-debt": {
```
Comment (vbstreetz) [inline translations/en.json:158]: revert

Comment (vbstreetz) [inline translations/en.json:158]: seems this is still being used

**Review by vbstreetz — CHANGES_REQUESTED** [review]

File: translations/en.json (hunk 1 above)
Comment (KillerByte) [reply translations/en.json:158]: fixed https://github.com/Synthetixio/stats/pull/134/commits/d567e313e2bcde2a4c702da7fd1b91f36cca98ef

File: sections/Staking/index.tsx [hunk 2]
```diff
@@ -71,9 +71,7 @@ function formatLiquidationsChart(
 			continue;
 		}
 
-		const value = Number(
-			ethers.utils.formatEther(ethers.BigNumber.from(liquidation.amountLiquidated))
-		);
+		const value = wei(liquidation.amountLiquidated).toNumber();
```
Comment (vbstreetz) [inline sections/Staking/index.tsx:74]: should this be divided by 1e18?

**Review by vbstreetz — CHANGES_REQUESTED** [review]

**Review by vbstreetz — APPROVED** [review]
Now looking good.

**Pull request merged**